### PR TITLE
Apm 1313 use ecs

### PR DESCRIPTION
**Pull request by Sourceless**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: azure/project.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 variables:
   service_name: identity-service
+  short_service_name: ids
   service_base_path: oauth2
   product_display_name: Identity Service
   product_description: Placeholder description
+  System.Debug: true
```
Comment (HenryIwanejko) [inline azure/project.yml:7]: Is this needed?

File: azure/azure-pr-pipeline.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ resources:
       type: github
       name: NHSDigital/api-management-utils
       endpoint: NHSDigital
+      ref: refs/heads/apm-1313-ecs-deploy-in-common-pipelines
```
Comment (HenryIwanejko) [inline azure/azure-pr-pipeline.yml:12]: Note: Change back to master before merge

**Comment by Sourceless** [comment]
Force merged because common pipelines are moving around

**Pull request merged**